aries-site/src/examples/templates/PageHeader/PageHeader.js

```diff
@@ -1,16 +1,65 @@
+import { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import { Header, Heading } from 'grommet';
+import { Box, Header, Heading, Paragraph } from 'grommet';
+import { FormPrevious } from 'grommet-icons';
 
-export const PageHeader = ({ title, actions, ...rest }) => (
-  <Header {...rest}>
-    <Heading level={1} margin="none" size="small">
-      {title}
-    </Heading>
-    {actions}
-  </Header>
-);
+import { UpButton } from './UpButton';
+import { BreadCrumbs } from './BreadCrumbs';
+
+export const PageHeader = ({
+  title,
+  subtitle,
+  actions,
+  contextualNav,
+  ...rest
+}) => {
+  const [upButton, setUpButton] = useState(null);
+  const [breadCrumbs, setBreadCrumbs] = useState(null);
```

Comment:
Where is breadcrumbs showing up on the page examples? I think I'm missing it

Reply:
If it's separate, maybe we should handle the PageHeader changes in its own PR

Reply:
Neither upButton nor breadCrumbs are in the examples. Was experimenting with how they might impact the page layouts. I can pull out of this PR.

Reply:
done.